crates/dbs-device/src/lib.rs

```diff
@@ -2,8 +2,6 @@
 // Copyright © 2019 Intel Corporation. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#![deny(missing_docs)]
```

Comment:
Why removing the `#![deny(missing_docs)]`?

Reply:
Sorry, I mistook it for `allow(missing_docs)`.
removed this change.